src/main/java/com/moodybluez/enterprise/dao/EntrySQLDAO.java

```diff
@@ -14,26 +17,28 @@
     @Autowired
     private EntryRepository entryRepository;
 
+    Logger LOG = LoggerFactory.getLogger(this.getClass());
+
     @Override
     public Entry save(Entry entry) {
         return entryRepository.save(entry);
     }
 
     @Override
     public List<Entry> fetchByMonth(int year, int month) {
-        List<Entry> t= entryRepository.findByMonth(year,month);
-        return t;
+        return entryRepository.findByMonth(year,month);
     }
 
     @Override
     public Entry fetchByDate(String date) {
         Date d = new Date();
+
         try {
             d = new SimpleDateFormat("yyyy-MM-dd", Locale.ENGLISH).parse(date);
+        } catch (ParseException e) {
+            LOG.error("Failed to Parse Date, please check if the date is in correct format. Date : " + date, e);
         }
-        catch (Exception e){
 
-        }
```

Comment:
Using Logger better logging of issues in program. It is generally a Best practice to use Logging rather than just System.out.println or even just leaving it blank.